public/Get-DbaMaintenanceSolutionLog.ps1

```diff
@@ -6,6 +6,8 @@ function Get-DbaMaintenanceSolutionLog {
     .DESCRIPTION
         Ola wrote a .sql script to get the content from the commandLog table. However, if LogToTable='N', there will be no logging in that table. This function reads the text files that are written in the SQL Instance's Log directory.
 
+        Be aware that this command only works if sqlcmd is used to execte the procedures, which is a legacy way to start them and not used by new installations.
```

Comment:
@andreasjordan  - typo - execte\execute is spelt wrong, also is it also worth noting here our recommendation? something like:

```suggestion
        Be aware that this command only works if sqlcmd is used to execute the procedures, which is a legacy way to start them and not used by new installations.

        We would instead recommend installing using Install-DbaMaintenanceSolution and calling the procedures with the LogToTable='Y' parameter.
```